### ISSUE TITLE
Add more surgeries - Act II

### ISSUE THREAD

**Issue by kaptain-kavern** [kaptain-kavern/ADogSaid#20]
This time i was thinking about recipe from [DESurgeries](https://github.com/kaptain-kavern/DESurgeries). They are "_specials_" because : 
- Some are meant to reconstruct the natural bodypart (opposing to attach an artificial bodypart like in ADS already
- Others are meant to treat an injury pre-emptively, before it infect or become permanent.
- And there also is treatment for all chronic diseases/illness
- Also an option for sedate a colonist to force him to get rest

I'm not sure you should include them all so i'm gonna paste the list and maybe you can discuss a bit what you think of the idea.

```
Adds new surgeries to Rimworld.

    Possibily to sedate patient
    2 cosmetic surgeries: Ear and Nose.
    8 chronic surgeries: Bad back, Frail, Cataract, Hearing loss, Dementia, Alzheimers, Asthma and Heart artery blockage.
    12 injury surgeries: Burn, Crush, Crack, Cut, Surgical cut, Scratch, Bite, Stab, Gunshot, Shredded, Bruise, Frostbite.
    2 Parasite surgeries : Muscle Parasite, Gut Worms
    8 bone/limb surgeries: Finger, Toe, Mandible, Pelvis, Clavicles, Sternum, Rib cage and Spine.
```

**Comment by inkyspoon** [comment]
ok so If we could reduce the clutter..or, at least not add to it, I think that's good. Of course not all those above surgeries are viable for animals. Cosmetic for example..Animals don't do social talking :)
Also I'm not sure about the chronic deseases, though I did see some with cataracts, asthma, and I think frail...The injuries, sure. Bone and limb, I'm pretty sure most aren't even defined for animals except spine.

But the question here is..some of those surgeries, like asthma, spine, catarct, would make the added replacement  that we have useless?

**Comment by nattiejim** [comment]
@spoonshortage - You're right about having surgeries to directly fix asthma, spines, and cataracts - it could very possibly make our replacements useless.

However, most of these other surgeries (injuries, chronic diseases) could prove very useful.

**Comment by kaptain-kavern** [comment]
When I play I use both DESurgeries and EPOE, and in a similar way we could think the same and say that DESurgeries make some bionics from EPOE useless. In fact I found that DESurgeries completed well EPOE as it take place at whole different level in term of cost in resource : most of the recipe "_only_" cost medicine (a lot of them) and it's balanced by the risk of death for the patient (up to 25% for the hardest treatment). So they're super handy early (but with the chance of success I've set you don't want to try a very hard surgery in a shitty hospital with an unskilled doctor) and bionics ate still valuable later in game because of the augmented stats

I was explaining this for the reflexion. We even can just do a extra patch to keep things separated ( or I can do this on DESurgeries side)
